pandas/io/parquet.py

```diff
@@ -277,6 +277,17 @@ def read_parquet(path, engine='auto', columns=None, **kwargs):
     Returns
     -------
     DataFrame
+        A parquet file is returned as two-dimensional data structure with
+        labeled axes.
+
+    See Also
+    --------
+    to_parquet : Write DataFrame to a parquet file.
```

Comment:
Does this get linked correctly by numpydoc / sphinx? Or does it need to be `DataFrame.to_parquet`?

Reply:
It appears it does need the `DataFrame.` prefix to link correctly. A brief bit of searching shows this problem exists on at least two other pages (both with `to_csv` in the See Also sections of `read_csv` and `read_fwf`). Is it better to commit a change for all three as part of this PR or just fix this one and create a new PR for the other two?

Reply:
Just fix this for now and leave the other two. Not sure if your check was only on this module but would be surprised if we only had two across the codebase, so would be fine to have a comprehensive PR separate to address

Reply:
```suggestion
    DataFrame.to_parquet : Write DataFrame to a parquet file.
```